css/_uswds-theme-custom-styles.scss

```diff
@@ -2998,68 +2979,10 @@ $theme-table-column-gap: 4;
         $site-banner-link-color,
         $context: $banner-context
       );
```

Comment:
**todo:** It looks like we can safely remove the `set-link-from-bg` mixin from lines 2977-2981 without visual regression.

Reply:
I removed the mixin and verified no visual regression. :)